clients/python/llmengine/model.py

```diff
@@ -167,6 +167,94 @@ def create(
                 An optional dictionary of key/value pairs to associate with this endpoint
         Returns:
             CreateLLMEndpointResponse: creation task ID of the created Model. Currently not used.
+
+        === "Create Llama 2 7B model in Python"
+            ```python
+            from llmengine import Model
+
+            response = Model.create(
+                name="llama-2-7b-test"
+                model="llama-2-7b",
+                inference_framework_image_tag="2.0.0",
```

Comment:
Is this a valid inference framework tag for vLLM? Thought this should be `"0.2.0"`? Though it looks like we may want to use `"0.2.1"` for a recent fix.